### PR TITLE
Update logs in withdraw_coin worker

### PR DESCRIPTION
**Pull request by dnfd**

### CODE REVIEW

File: app/workers/worker/withdraw_coin.rb [hunk 1]
```diff
@@ -6,70 +6,85 @@ class WithdrawCoin
...
 
         balance = wallet.current_balance
         if balance == 'N/A' || balance < withdraw.amount
-          Rails.logger.warn do
-            "The withdraw skipped because wallet balance is not sufficient or amount greater than wallet max_balance"\
-            "wallet balance is #{balance.to_s}, wallet max balance is #{wallet.max_balance.to_s}."
-          end
+          Rails.logger.warn worker: 'Withdraw coin', id: withdraw.id,
+                            balance: balance.to_s,
+                            max_balance: wallet.max_balance.to_s,
```
Comment (mod) [inline app/workers/worker/withdraw_coin.rb]: remove

File: app/workers/worker/withdraw_coin.rb [hunk 2]
```diff
@@ -6,74 +6,88 @@ class WithdrawCoin
     def process(payload)
       payload.symbolize_keys!
 
-      Rails.logger.warn { ">>>>> Received request for processing withdraw ##{payload[:id]}." }
+      Rails.logger.warn worker: __FILE__, id: payload[:id], message: 'Received request for processing withdraw.'
```
Comment (ysv) [inline app/workers/worker/withdraw_coin.rb]: 1. You can initialize logger once with class name and use it e.g:
```ruby
logger = DaemonsLogger.new(self.class.name)

logger.warn id: payload[:id], message: "Something"
```

2. Or we can inherit all AMQP daemons from Base class and define logger method here

I prefer the second way

**Review by ysv — CHANGES_REQUESTED** [review]

File: app/workers/amqp/withdraw_coin.rb [hunk 3]
```diff
@@ -4,77 +4,95 @@
...
 
           withdraw.txid = transaction.hash
           withdraw.dispatch
           withdraw.save!
 
-          Rails.logger.warn { "OK." }
+          @logger.warn id: withdraw.id, message: 'OK.'
 
-        rescue Exception => e
+        rescue StandardError => e
```
Comment (ysv) [inline app/workers/amqp/withdraw_coin.rb]: keep `Exceptions` here we will update it in separate patch

File: lib/peatio/tagged_logger.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    @logger = logger.dup
+    @tags = tags
+  end
+
+  [:fatal, :error, :warn, :info, :debug].each do |log_method|
+    define_method(log_method) do |msg|
+      if msg.is_a? Hash
+        msg = @tags.merge msg
+      else
+        msg = "#{@tags.map { |k, v| "#{k}: #{v}" }.join(', ')}. #{msg}"
```
Comment (ysv) [inline lib/peatio/tagged_logger.rb]: @denisfd if it's a string use 
```@tags.merge(message: msg)```

**Review by ysv — CHANGES_REQUESTED** [review]

**Review by mnaichuk — APPROVED** [review]

**Review by ysv — APPROVED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**